frontend/src/components/contactsForm.tsx

```diff
@@ -0,0 +1,310 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
let's avoid these file level ignores - either configure it in the eslint config or localize to only the offending line

Reply:
removed